src/routes/view-content/library-resource-menu/LibraryResourceMenu.svelte

```diff
@@ -133,7 +133,7 @@
                 <button
                     on:click={(e) => resetPage(e)}
                     type="button"
-                    class="mt-2 ps-4 font-semibold text-primary"
+                    class="mt-2 pl-4 font-semibold text-primary"
```

Comment:
Looks like somehow this got switched back to `pl`.

Reply:
Wow that is so bizarre, good catch